### PR TITLE
Added armv7hl image for mageia 6

### PR DESCRIPTION
**Pull request by juanluisbaptiste**
Hi,

We have added support for armv7hl to the mageia 6 image, let me know if there's any mistake with the file format.

### CODE REVIEW

File: library/mageia [hunk 1]
```diff
@@ -2,6 +2,10 @@ Maintainers: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
...
-GitCommit: f60e1b20a5e31e72b4f49c878160dfa95f49a911
-GitFetch: refs/heads/dist
-Directory: 6
+Architectures: amd64, arm32v7
+amd64-GitCommit: e7ffd1ff77c5914b7f46a6cee2373acfb0bf60f5
+amd64-Directory: 6/x86_64
+amd64-GitFetch: refs/heads/dist
+arm32v7-GitCommit: e7ffd1ff77c5914b7f46a6cee2373acfb0bf60f5
+arm32v7-GitFetch: refs/heads/dist
+arm32v7-Directory: 6/armv7h1
```
Comment (Conan-Kudo) [inline library/mageia]: This is wrong, it should be `6/armv7hl`, right?

Comment (juanluisbaptiste) [reply library/mageia]: Ahhhhhhhhh I was starting to pull my beard out on whats wrong, good catch !!

File: library/mageia [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+amd64-GitFetch: refs/heads/dist
+arm32v7-GitCommit: e7ffd1ff77c5914b7f46a6cee2373acfb0bf60f5
+arm32v7-GitFetch: refs/heads/dist
+arm32v7-Directory: 6/armv7hl
 
 Tags: 6, latest
-GitCommit: f60e1b20a5e31e72b4f49c878160dfa95f49a911
-GitFetch: refs/heads/dist
-Directory: 6
+Architectures: amd64, arm32v7
```
Comment (tianon) [inline library/mageia]: Here's what I'd recommend:

```
Maintainers: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
GitRepo: https://github.com/juanluisbaptiste/docker-brew-mageia.git
GitCommit: e7ffd1ff77c5914b7f46a6cee2373acfb0bf60f5
GitFetch: refs/heads/dist

Tags: 6, latest
Architectures: amd64, arm32v7
amd64-Directory: 6/x86_64
arm32v7-Directory: 6/armv7hl
```

Comment (juanluisbaptiste) [reply library/mageia]: Let my try this...

**Comment by tianon** [comment]
Diff: https://gist.github.com/docker-library-bot/b7fa1674279e44e5e2e1bea5cbba05ed

(Appears to be adding a bit more by default -- you might want to double check that the way you're generating these is including the minimal set of packages you expect it to be.)

LGTM

Build test of #5886; 3d438773; `arm32v7` (`mageia`):

```console

$ bashbrew build mageia:6
Building bashbrew/cache:dd7339b7dd178a482fe2e2fe770e87ea47dc6f88583227a12804400f4ca9f85a (mageia:6)
Tagging mageia:6
Tagging mageia:latest

$ test/run.sh mageia:6
testing mageia:6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

Do you want to merge this as-is and double check that diff separately, or would you rather wait until you have a chance to verify the diff?

**Comment by juanluisbaptiste** [comment]
@Conan-Kudo WDYT ? I will review and test amd64 build but I don't have how to test the armv7 build.

**Comment by juanluisbaptiste** [comment]
@tianon I think you can merge it as is while we take a closer look at the default package list.

**Comment by Conan-Kudo** [comment]
@juanluisbaptiste The additions are intentional.

@tianon However, I don't understand why this diff produced by the bot says that the we lost x86_64 and added armv7hl. What happened to the x86_64 image?

**Comment by tianon** [comment]
It's just showing the difference between the old amd64 tarball and the new
arm32v7 tarball.

**Comment by Conan-Kudo** [comment]
@tianon So we're not going to lose the x86_64 (amd64) container image? As long as that's true, then this should be all set.

**Comment by tianon** [comment]
Correct -- this doesn't actually change the amd64 tarball as far as I can tell.  Merging.

**Comment by juanluisbaptiste** [comment]
@tianon thanks !! when will the armv7 image be available in docker hub ?

**Comment by tianon** [comment]
It should already be! :smirk: :heart:

**Comment by juanluisbaptiste** [comment]
I ask because I still don't see the armv7 tag on the image description at docker hub...

**Comment by tianon** [comment]
Ah right, the description should be updated within the next 15 minutes or so. :+1:

**Comment by tianon** [comment]
```console
$ manifest-tool inspect mageia:6
Name:   mageia:6 (Type: application/vnd.docker.distribution.manifest.list.v2+json)
Digest: sha256:930cc11d102f98a5597a62428f708c0e1d12ae595f51451494159e9776a70876
 * Contains 2 manifest references:
1    Mfst Type: application/vnd.docker.distribution.manifest.v2+json
1       Digest: sha256:b4463430781037c19f4880fee581747d872b32d577bd9cb61e0ffd512b42816f
1  Mfst Length: 530
1     Platform:
1           -      OS: linux
1           - OS Vers: 
1           - OS Feat: []
1           -    Arch: amd64
1           - Variant: 
1           - Feature: 
1     # Layers: 1
         layer 1: digest = sha256:6b0f0e2f5c4179bfb477d676d1aee0deed41b4c6a874fc1b9e4167683673ca2c

2    Mfst Type: application/vnd.docker.distribution.manifest.v2+json
2       Digest: sha256:bfc8f0379607ad4005b4d6b84257b0751753d0218ec3521e3b8ac9acd3535c3a
2  Mfst Length: 530
2     Platform:
2           -      OS: linux
2           - OS Vers: 
2           - OS Feat: []
2           -    Arch: arm32
2           - Variant: v7
2           - Feature: 
2     # Layers: 1
         layer 1: digest = sha256:c04d8d343c6f1599282e7140af6b9aa572ee838b3aec1565f8d6052a37f23a1f

```

and:

```console
$ docker pull arm32v7/mageia:6
6: Pulling from arm32v7/mageia
c04d8d343c6f: Pull complete 
Digest: sha256:bfc8f0379607ad4005b4d6b84257b0751753d0218ec3521e3b8ac9acd3535c3a
Status: Downloaded newer image for arm32v7/mageia:6
```

**Pull request merged**